rskj-core/src/test/java/org/ethereum/net/rlpx/NodeTest.java

```diff
@@ -89,4 +90,55 @@ void getAddress() {
 
     }
 
+    @Test
+    void nodeCreationWithUrlIsParsedCorrectly() {
+        String host = "host";
+        int port = 1234;
+        String hexNodeId = String.valueOf(Hex.encodeHex(NODE_ID_1));
+        String enodeUrl = "enode://" + hexNodeId + "@" + host + ":" + port;
+        Node node = new Node(enodeUrl);
+        assertNotNull(node);
+
+        assertEquals(port, node.getPort());
+        assertEquals(host, node.getHost());
+        assertEquals(hexNodeId, node.getHexId());
+    }
+
+    @Test
+    void nodeCreationWithBadSchemeMustFail() {
+        assertThrows(RuntimeException.class, () -> new Node("http://www.google.es"));
```

Comment:
WDYT about validating the message is the expected one? `RuntimeException` is quite generic and could happen internally on libraries (although it shouldn't, we know it happens)

Maybe also modifying a bit the message on each case that it's thrown in the constructor to differentiate and validate on tests.